mite/context.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import time
 import traceback
+from contextvars import ContextVar
```

Comment:
...not...in...alphabetical...order... 😅 

![obsessive](https://user-images.githubusercontent.com/185075/119861661-8adbb000-bf0f-11eb-8745-f4b175a80d0d.gif)